docs/algorithms.rst

```diff
@@ -347,27 +340,231 @@ This speeds up mixing and reduces the total number of samples you need to get un
+
+In practice, if the initial conditions are very atypical of equilibrium (which is often the case in molecular
+simulation, especially after energy minimization), it helps a great deal to discard an initial part of the simulation to
+equilibration. But how much? How do we decide? YANK chooses an automatic equilibrium detection scheme based on the
+following articles: :cite:`Shirts2008` :cite:`Chodera2011` :cite:`Chodera2016`
+
+Determining a timeseries to analyze in a replica-exchange simulation
+--------------------------------------------------------------------
+
+For a standard molecular dynamics simulation producing a trajectory :math:`x_t`, it's reasonably straightforward to
+decide approximately how much to discard if human intervention is allowed. Simply look at some property
+:math:`A_t = A(x_t)` over the course of the simulation. This property should ideally be one that we know has some slow
+behavior that may affect the quantities we are interested in computing and find the point where :math:`A_t` seems to
+have "settled in" to typical equilibrium behavior. :math:`A(x)` is a good choice if we're
+interested in the expectation :math:`\left\langle A \right\rangle`.
+
+If we're interested in a free energy, which is computed from the potential energy differences,
+let's suppose the potential energy :math:`U(x)` may be a good quantity to examine.
+
+In replica-exchange simulation, there are K replicas that execute nonphysical walks on many potential energy
+functions :math:`U_k(x)`. The multiple states call into question what observable should be examined.
+Let's work by analogy. In a single simulation, we would plot some quantity related to the potential energy :math:`U(x)`,
+or its reduced version :math:`u(x) = \beta U(x)`. This is actually the negative logarithm of the probability
+density :math:`\pi(x)` sampled, up to an additive constant:
+
+.. math::
+
+    u(x) = -\ln \pi(x) + c
+
+For a replica-exchange simulation, the sampler state is given by the pair :math:`(X,S)`, where
+:math:`X = \{x_1, x_2, \ldots, x_K \}` are the replica configurations and :math:`S = \{s_1, s_2, \ldots, s_K\}` is the
+vector of state index permutations associated with the replicas. The total probability sampled is
+
+.. math::
+
+    \Pi(X,S) &= \prod_{k=1}^K \pi_{s_k}(x_k) \\
+             &= (Z_1 \cdots Z_K) \exp\left[-\sum_{k=1}^K u_{s_k}(x_k)\right] \\
+             &= Q^{-1} e^{-u_*(X)}
+
+where the pseudoenergy :math:`u_*(X)` for the replica-exchange simulation is defined as
+
+.. math::
+
+    u_*(X) \equiv \sum_{k=1}^K u_{s_k}(x_k)
+
+That is, :math:`u_*(X)` is the sum of the reduced potential energies of each replica configuration at the current
+thermodynamic state it is visiting.
+
+Determining equilibration and autocorrelation in a replica-exchange simulation
+------------------------------------------------------------------------------
+
+Now that we have the observable, we now need to evaluate the equilibration, and autocorrelation times.
```

Comment:
Suggest: "YANK automatically determines an optimal partitioning between the initial equilibration phase of the simulation (which is discarded) and the production phase of the simulation (which is analyzed to estimate free energies)."